tests/Feature/Drivers/GoogleCalendarDriverTest.php

```
@@ -151,6 +151,102 @@ public function accessTokenFor( CalendarConnection $connection ): string
+    } );
+
+    it( 'surfaces the reason on a refused delete, list, watch, and sync too', function (): void {
+        Http::fake( [ 'https://www.googleapis.com/*' => Http::response( accessNotConfiguredBody(), 403 ) ] );
+
+        $connection = CalendarConnection::factory()->google()->twoWay()->create( [ 'sync_token' => 'token-here' ] );
+        $window     = new TimeRange(
+            CarbonImmutable::parse( '2026-06-01T00:00:00Z' ),
+            CarbonImmutable::parse( '2026-06-02T00:00:00Z' ),
+        );
+
+        expect( fn (): mixed => googleDriver()->deleteEvent( $connection, 'apbk-whatever' ) )
```

PHPStan (2.2.7)

[error] Anonymous function with return type void returns null but should not return anything.


(return.void)